core/sqf/sql/scripts/sqstart

```diff
@@ -245,6 +245,39 @@ function checkKerberos {
    return 0
 }
 
+function checkUlimit {
+  
+ #check max_pid 
+ userProc=`cat /proc/sys/kernel/pid_max`
+ if [[ $userProc -gt 65535 ]] ; then
+    echoLog ""
+    echoLog "ERROR: detect issue during startup, please make sure your kernel parameter (sysctl) setup correctly"
+    echoLog "ERROR: max_pid should not be greater than 65535"
+    echoLog ""
+    return
+ fi
+
+ #check ulimit -l
+ maxLockMem=`ulimit -l`
+ if [[ $maxLockMem -lt 65536 ]] ; then
+    echoLog ""
+    echoLog "ERROR: detect issue during startup, please make sure your ulimit setup correctly"
+    echoLog "ERROR: max locked memory is smaller than 64M"
+    echoLog ""
+    return 
+ fi
+  
+ #check ulimit -l
+ if [[ $maxLockMem -lt 327680 ]] ; then
+    echoLog ""
+    echoLog "WARNING: detect issue during startup, please make sure your ulimit setup correctly"
```

Comment:
hi, Dave, I think this is ulimit setting. 'max lock memory' can be set via 'ulimit -l'.  It is same number check as above. If it is smaller than 64M, it is an Error, if it is bigger than 64M, but smaller than 320M, it is a Warning, as suggested by @selvaganesang . 
But I will update the words with what you mentioned below.